src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -361,6 +361,7 @@ AutomationInstrumentClipView::AutomationInstrumentClipView() {
 	leftPadSelectedY = kNoLastSelectedPad;
 	rightPadSelectedX = kNoLastSelectedPad;
 	rightPadSelectedY = kNoLastSelectedPad;
```

Comment:
create a new variable that remembers the knob position value for the last pad selected. This will be used in case the value pop-up needs to be re-drawn (e.g. if you press an audition pad and it hides the value popup by drawing note code / drum name, this will be used when the timer kicks in to re-draw the value for the pad selected).